system_tests/systemd

```diff
@@ -35,7 +35,7 @@ mkdir -p /home/ubuntu/.ssh
 chown -R ubuntu:ubuntu /home/ubuntu/.ssh
 
 mkdir -p /opt/parallelcluster
-echo "aws-parallelcluster-cookbook-3.0.1" > /opt/parallelcluster/.bootstrapped
+echo "aws-parallelcluster-cookbook-3.1.0b1" > /opt/parallelcluster/.bootstrapped
```

Comment:
what about adding this logic to the util/bump_version.sh script so that this version gets updated every time we bump the package version?

Reply:
resolved here: https://github.com/aws/aws-parallelcluster-cookbook/pull/1296/commits/e67a98b6b61f2fad35322961ca42e8ba4752da5a